src/v2/Components/AstroChat/useAstro.ts

```diff
@@ -69,6 +69,7 @@ const loadMessage = async (
         message.options = resolvedContent.options?.map((b) => ({
           value: b.value,
           text: b.text,
+          optionId: b.option_id,
```

Comment:
Adding `option_id` here and there - This is a new api feature so we can target suggestions directly instead of only free text.